### PR TITLE
feat(elements|ino-icon): add css variable for the rotation of icons

### LINKED ISSUE #332: Rename icon file names
Currently we still have some icons that contain a German word. We should change that.

**Acceptance criteria**

- [x] The following icon names have been changed:
  * status_absent => employment_status_away
  * status_offboarding_ongoing => employment_status_offboarding
  * status_onboarding_ongoing => employment_status_onboarding
  * status_current = employment_status_ongoing
  * status_past = employment_status_ended
  * status_future = employment_status_starts_soon
- [x] The code does not contain 10 if / else clauses
- [x] Tests have been implemented, if applicable
- [x] Fallback fold the old names exists and has been tested.

### LINKED ISSUE #363: ino-icon: add arrow icons for every direction
### Is your feature request related to a problem? Please describe.

We've got the `back_arrow` icon in our library, which can also indicate directions. Therefore, we should add arrows pointing in every direction (forward, upward, downward) due to consistency reasons.

### Describe the solution you'd like

Use the `back_arrow` icon as a base, rotate them and add the icons as `forward_arrow`, `upward_arrow`, `downward_arrow`.
The names are just a suggestion.

### PR DESCRIPTION
**Pull request by P1NHE4D**
✨ **Features:**
* Added `--ino-icon-rotation` CSS variable to allow for the rotation of icons

🔧 **Fixes:**
* Fixed an `eslint` error caused by the incorrect configuration in `lint-staged`

⚠️ **Breaking changes:**
* Removed `arrow_left`, `arrow_down`, and `arrow_up` icons
* Renamed `arrow_right` icon to `arrow`
* Renamed `status_abwesend`, `status_aktuelle`, `status_offboarding_laufend`, `status_onboarding_laufend`, `status_vergangene`,  and `status_zukuenftige` to `employment_status_away`, `employment_status_ongoing`, `employment_status_offboarding`, `employment_status_onboarding`, `employment_status_ended`, and `employment_status_starts_soon` respectively according to #332

🎟️ **Closes #363**, **Closes #332**

### CODE REVIEW

File: .lintstagedrc [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "!(*eslintrc).{t,j}s{,x}": ["prettier --write", "eslint --fix"],
+  "!(*eslintrc|*.stories).{t,j}s{,x}": ["prettier --write", "eslint -c packages/elements/.eslintrc.js --fix"],
```
Comment (BenPag) [inline .lintstagedrc:2]: Can you explain these changes?

**Comment by BenPag** [comment]
In fact of breaking changes we need to check if other coponenten use the old arrow icon internally. E. g. `ino-fab-set`

File: .lintstagedrc (hunk 1 above)
Comment (janivo) [reply .lintstagedrc:2]: I think it was meant as a workaround to the linting problem we had before. We had no lint-Files in our storybook package was lead to errors when trying to commit. This got fixed in https://github.com/inovex/elements/pull/410.

**Comment by pfecht** [comment]
Closed for now, will be addressed in future if necessary.

**Pull request closed without merging**